File: tests/complex/fizzbuzz.swift
public func fizzBuzz() -> [String] {
    return (1...100).map { (num: Int) -> String in return num % 3 == 0 ? num % 5 == 0 ? "FizzBuzz" : "Fizz" : num % 5 == 0 ? "Buzz" : String(num) };
}

File: tests/complex/fizzbuzz.js
export function fizzBuzz() {
  const mapped = [];

  for (let i = 1; i <= 100; i++) {
    mapped.push(i % 3 === 0 ? i % 5 === 0 ? "FizzBuzz" : "Fizz" : i % 5 === 0 ? "Buzz" : String(i));
  }

  return mapped;
}